pkgs/development/haskell-modules/configuration-ghc-8.2.x.nix

```diff
@@ -71,4 +71,12 @@ self: super: {
   # https://github.com/aristidb/aws/issues/238
   aws = doJailbreak super.aws;
 
+  # There's no 8.2 testsuite
+  ghc-exactprint = dontCheck super.ghc-exactprint;
+
+  # Missing FlexibleContexts in test suite
+  monad-memo = dontCheck super.monad-memo;
+
+  haddock-api = super.haddock-api.override
```

Comment:
Is this override specific to GHC 8.2.2? It looks like it might belong into `configuration-nix.nix` or `configuration-common.nix`.

Reply:
Right, I confused myself about the order of overrides, I didn't want to overwrite the old versions uses in 7.10 and 7.8. Will fix.